help.go

```diff
@@ -472,9 +472,17 @@ func formatFlag(haveShort bool, flag *Flag) string {
 		flagString += fmt.Sprintf("-%c, --%s", flag.Short, name)
```

Comment:
I think it will need to go here too? And maybe refactor to remove the duplication?

Reply:
@alecthomas done, updated